test/Microsoft.SqlTools.Migration.IntegrationTests/Migration/MigrationServiceTests.cs

```diff
@@ -58,7 +58,8 @@ public async Task TestHandleMigrationGetSkuRecommendationsRequest()
                 EndTime = DateTime.UtcNow.ToString("yyyy-MM-dd HH:mm:ss"),
                 PerfQueryIntervalInSec = 30,
                 ScalingFactor = 1,
-                DatabaseAllowList = new List<string> { "test", "test1" }
+                DatabaseAllowList = new List<string> { "test", "test1" },
+                IsPremiumSSDV2Enabled = true,
```

Comment:
You might want to look into adding tests for when it is false.

Reply:
Never mind. The test is ignored.

Reply:
False is just to make it backward compatible, which is any way ignored. :)